api/infrastructure/parallelcluster-api.yaml

```diff
@@ -59,6 +59,16 @@ Parameters:
       - true
       - false
 
+  EnableS3FullAccess:
+    Description: |
+      When set to true the ParallelCluster API can access, write all the S3 buckets, it is needed to import/export from/to S3 when creating an Fsx filesystem.
+      WARNING - setting this to true grants to the Lambda function S3 Get*, List* and PutObject privileges on all the buckets
```

Comment:
Minor: missing full stop at end of line.

Reply:
Done